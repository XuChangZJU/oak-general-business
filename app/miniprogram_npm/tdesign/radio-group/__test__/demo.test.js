import simulate from 'miniprogram-simulate';
import getDemoPath from '../../../test/utils/getDemoPath';

describe('radio group demo', () => {
  let id;
  beforeAll(() => {
    id = simulate.load(getDemoPath('radio', 'group'), {
      less: true,
    });
  });
  test('demo should render correctly', async () => {
    const container = simulate.render(id);
    container.attach(document.createElement('parent-wrapper'));
    expect(container.toJSON()).toMatchSnapshot();
  });
});
