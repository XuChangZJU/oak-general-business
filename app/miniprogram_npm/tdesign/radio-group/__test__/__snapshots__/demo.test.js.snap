// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`radio group demo demo should render correctly 1`] = `
Object {
  "attrs": Array [],
  "children": Array [
    Object {
      "attrs": Array [
        Object {
          "name": "class",
          "value": "radio-group",
        },
        Object {
          "name": "value",
          "value": "",
        },
      ],
      "children": Array [],
      "event": Object {
        "change": Object {
          "handler": "onChange",
          "id": 1,
          "isCapture": false,
          "isCatch": false,
          "isMutated": false,
          "name": "change",
        },
      },
      "tagName": "t-radio-group",
    },
  ],
  "event": Object {},
  "tagName": "main",
}
`;
